docs/contributors/triage.md

```diff
@@ -18,33 +20,33 @@ These are the expectations of being a triage team member:
 
 If you would like to join this team, simply ask in #core-editor [Slack](https://make.wordpress.org/chat/) at any time.
 
-### Getting started
+## Triage on your own
```

Comment:
This doesn't feel as welcoming as "Getting started", which is fairly common for these kinds of docs. I might rename to something else or keep as is: "Triage your first issues" perhaps.

Reply:
Yes, "Triage your first issues" sounds better 😊
Fixed!